_api-reference/cat/cat-repositories.md

```diff
@@ -29,7 +29,7 @@ GET _cat/repositories
 
 All cat repositories URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameters:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameters:
```

Comment:
`[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)`